### ISSUE TITLE
[Question] about mongo db storage key index

### ISSUE THREAD

**Issue by mesaks** [gofiber/storage#411]
hello, while examining mongodb storage, I noticed that it is querying with the key field, but this field is not indexed.

Is this just something that was overlooked? Or is it not needed?

Thanks.

**Comment by gaby** [comment]
@mesaks You are right, this may hurt performance when a lot of data is sent to the database. I will be submitting a PR to fix this.
